archinstall/default_profiles/desktops/cosmic.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, Optional, Any, TYPE_CHECKING
+
+from archinstall.default_profiles.profile import ProfileType, GreeterType
+from archinstall.default_profiles.xorg import XorgProfile
+
+if TYPE_CHECKING:
+	_: Any
+
+class PlasmaProfile(XorgProfile):
+	def __init__(self):
+		super().__init__('cosmic-epoch', ProfileType.DesktopEnv, description='')
+
+	@property
+	def packages(self) -> List[str]:
+		return [
+			"cosmic",
+		]
+
+	@property
+	def default_greeter_type(self) -> Optional[GreeterType]:
+		return GreeterType.cosmic-session
```

Comment:
And dashes is not valid variable names, consider `GreeterType.cosmic_session` after adding that greeter here: https://github.com/archlinux/archinstall/blob/b0f2e446398d17f39fe8079f1cd6ff970a0bd86a/archinstall/default_profiles/profile.py#L30-L35

And if no greeter is needed, you can omit the whole function:
```diff
-	@property
-	def default_greeter_type(self) -> Optional[GreeterType]:
-		return GreeterType.cosmic-session
```

Just like in the [awesome](https://github.com/archlinux/archinstall/blob/b0f2e446398d17f39fe8079f1cd6ff970a0bd86a/archinstall/default_profiles/desktops/awesome.py) desktop profile.